### ISSUE TITLE
Remove current transaction status when switching chains.

### ISSUE THREAD

**Issue by tomusdrw** [paritytech/parity-bridges-ui#157]
Related: #66 

#140 fixes the switch, but I think we shouldn't be displaying active transaction when chains are changed at all. Displaying some old transaction status below a UI that allows you to compose a new one (on a completely different chain, with different properties) is IMHO very misleading and confusing.

I'm fine if after the switch, the transaction status is only accessible from the history drawer, but only if we are connected back to the same source chain (transaction history should be per-source-chain, not "global"). We can indicate that there are transactions pending on other chains or sth, if we want to be fancy.
Another option is that if we switch back the transaction is still displayed - basically we can just always display last transaction from history below the main UI, but I'm not 100% sure if it would be that ergonomic. 

Wider discussion: https://github.com/paritytech/parity-bridges-ui/pull/140#discussion_r630134362

Curious also to hear @goldsteinsveta thoughts on this.

**Comment by goldsteinsveta** [comment]
> Displaying some old transaction status below a UI that allows you to compose a new one (on a completely different chain, with different properties) is IMHO very misleading and confusing

Imo Transactions have to to go from the main UI form as explained in #108 

> transaction history should be per-source-chain, not "global" 

That's like **My** Transactions, rather than block explorer though. For the drawer I can't see a reason to hide anything, there's an indication of what's source and what's target on the status card. Selecting a different sender to swap bridge to see the history is a bit bumpy ux too. Is global history technically complicated? Will disabling live status update help it? Will storing transactions in local storage help managing the history?

> we can just always display last transaction from history below the main UI, but I'm not 100% sure if it would be that ergonomic.

This can be a tmp fix 100%

**Comment by tomusdrw** [comment]
> Imo Transactions have to to go from the main UI form as explained in #108

Yeah, the behavior for current transaction status is perfectly described in #108. I 100% support this approach. Note that it also fixes the issue of switching, since it is now not possible to switch at all.

> That's like My Transactions, rather than block explorer though. For the drawer I can't see a reason to hide anything

I didn't study the designs of the drawer that much yet or even think how this could work, but I can see this being implemented indeed, we just need to be super clear with indicating what bridge the transaction is related too.

However I think this is only imported in the long-term future, when we support multiple different bridges. Let me elaborate why.
Given that we plan to have one bridge for now, I expect the users will be **only** interacting with a particular chain. Like I don't think they will casually switch between source and target chains, cause why would be they using the bridge if they have accounts on both sides anyway (given that we don't offer that much functionality yet)?

I find introducing global history a pretty significant development effort to do 100% right without introducing messy code (see #159 for instance), so I'd rather have more "block explorer" like experience. I imagine we can display a CTA, like "See your Milllau transactions (5)." at the end, since we do know this information (the `localStorage` is global already), but clicking this link would simply switch the source chain for you.
